__init__.py

```diff
@@ -30,14 +34,39 @@ def runtime_requirements(self):
             no_network_fallback=True,
             no_gui_fallback=True,
         )
+    
+    def initialize(self):
+        """Performs any final setup of the Skill, for instance to register
+        handlers for events that the Skill will respond to.
+        This is a good place to load and pre-process any data needed by your Skill.
+        """
+        # This initializes a settings dictionary that the skill can use to
+        # store and retrieve settings. The skill_settings.json file will be
+        # created in the location referenced by self.settings_path, which
+        # defaults to ~/.config/mycroft/skills/<skill_id>
+        # only new keys will be added, existing keys will not be overwritten
+        self.settings.merge(DEFAULT_SETTINGS, new_only=True)
+        # set a callback to be called when settings are changed
+        self.settings_change_callback = self.on_settings_changed
+        # (custom) event handler setup example
+        # below is a custom event, system event specs found at
+        # https://github.com/OpenVoiceOS/message_spec/tree/master/docs
```

Comment:
maybe link the rendered website instead? https://openvoiceos.github.io/message_spec/

Reply:
have you clicked the link? 🙈

Reply:
Forget what i've said, this is standalone and seperated from the rest of the documentation.
I fingured it is not rendering at all :P